Symfony/CS/Tests/Fixtures/Integration/misc/php_closing_tag,short_tag.test

```diff
@@ -0,0 +1,15 @@
+--TEST--
+Integration of fixers: php_closing_tag,short_tag.
+--CONFIG--
+level=none
+fixers=php_closing_tag,short_tag
+--REQUIREMENTS--
+hhvm=false
+--INPUT--
+<?
+$a = 1;
+
+?>
+--EXPECT--
+<?php
+$a = 1;
```

Comment:
Then it won't pass.
And I don't want to add extra fixer to this test configuration just to have that line.